CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## v0.10.0
 
-- Add support for Snapdragon 855
+- Added support for Snapdragon 855
+- Improved `VisionView` lifecycle management
+- Updated API of `VisionManager`
 
```

Comment:
shouldn't we add `New segmentation model recognizes...` as in [`ios`](https://github.com/mapbox/vision-ios-examples/blob/master/CHANGELOG.md) ?